Gems/AtomLyIntegration/CommonFeatures/Code/Source/SpecularReflections/SpecularReflectionsComponentConfig.cpp

```diff
@@ -0,0 +1,105 @@
+/*7
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <SpecularReflections/SpecularReflectionsComponentConfig.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/EditContext.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        void SpecularReflectionsComponentSSRConfig::Reflect(ReflectContext* context)
+        {
+            SSROptions::Reflect(context);
+
+            if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+            {
+                serializeContext->Class<SpecularReflectionsComponentSSRConfig>()
+                    ->Version(1)
+                    ->Field("Options", &SpecularReflectionsComponentSSRConfig::m_options)
+                ;
+
+                if (AZ::EditContext* editContext = serializeContext->GetEditContext())
+                {
+                    editContext->Class<SSROptions>("Screen Space Reflections (SSR)", "Screen Space Reflections (SSR) Configuration")
+                        ->ClassElement(Edit::ClassElements::EditorData, "")
+                        ->DataElement(Edit::UIHandlers::CheckBox, &SSROptions::m_enable, "Enable SSR", "Enable Screen Space Reflections (SSR)")
+                            ->Attribute(AZ::Edit::Attributes::ChangeNotify, Edit::PropertyRefreshLevels::EntireTree)
+                        ->DataElement(Edit::UIHandlers::Slider, &SSROptions::m_maxRayDistance, "Maximum Ray Distance", "The maximum length of the rays to consider for hit detection")
+                            ->Attribute(AZ::Edit::Attributes::Visibility, &SSROptions::IsEnabled)
+                            ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                            ->Attribute(AZ::Edit::Attributes::Max, 100.0f)
+                            ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                        ->DataElement(Edit::UIHandlers::Slider, &SSROptions::m_maxDepthThreshold, "Maximum Depth Threshold", "The maximum delta between the ray depth and depth buffer value which will be considered a hit.  Also known as thickness.")
```

Comment:
Is 'thickness' common enough terminology in other tools that it would be worth calling this "Maximum Depth Threshold (Thickness)" on the component card, instead of just in the tooltip?